### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures select element has an accessible name (select)

### ISSUE THREAD

**Issue by luisquintanilla** [luisquintanilla/insightsdemo#2]
#### Issue
`Ensures select element has an accessible name` ([select-name](https://dequeuniversity.com/rules/axe/4.4/select-name?application=msftAI))
#### Target application
[Welcome to CityLights! [Inaccessible Home Page]](https://www.w3.org/WAI/demos/bad/before/home.html)
#### Element path
select
#### Snippet
`<select onchange="location.href = this.value;">`
#### How to fix
    Fix any of the following:
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
#### Environment
Microsoft Edge version 101.0.1210.47
This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
